tests/_utils/wait_process_exit

```diff
@@ -1,15 +1,35 @@
 #!/bin/bash
 # parameter 1: process name
-
 process=$1
 
-while true; do
-	pgrep $process >/dev/null 2>&1
-	ret=$?
-	if [ "$ret" != "0" ]; then
+# Number of time to wait before using "kill -9"
+WAIT_COUNT=3
+
+# Counter to keep count of how many seconds have passed
+count=0
+
+PID=$(pgrep $process)
+if [ -z "$PID" ]; then
+	echo "process $process already exit"
+	exit
+fi
+
+while kill $PID >/dev/null; do
```

Comment:
Do we actually need to send kill signal multi times? I think kill once and wait in a loop is enough

Reply:
fixed